src/main/webapp/WEB-INF/tags/instructor/results/noResponsePanel.tag

```diff
@@ -27,7 +27,12 @@
                 <tbody>
                     <c:forEach items="${noResponsePanel.emails}" var="email">
                         <tr>
-                            <td>${fn:escapeXml(noResponsePanel.teams[email])}</td>
+                            <c:if test="${fn:escapeXml(noResponsePanel.teams[email])  == 'Instructors'}">
```

Comment:
A problem with this approach is that it fails to distinguish between actual instructors and students which have the team name "Instructors", which the old behaviour actually accounted for.